src/components/store/CollectionSlider/index.tsx

```diff
@@ -1,32 +1,60 @@
 import { Carousel } from '@/components/common';
+import CreateButton from '@/components/store/CreateButton';
```

Comment:
since this is also a store component, there will be a dependency circle because `@/components/store` already depends on `@/components/CollectionSlider`